src/test/resources/pages/instructorFeedbackSubmitPagePrivate.html

```diff
@@ -52,6 +52,7 @@
               </label>
               <div class="col-sm-10">
                 <p class="form-control-static">
+                  ${datetime.now} UTC+0800
```

Comment:
Or modify `convertLocalDateToUtc` to return null if input date is null? This seems to be consistent throughout `TimeHelper`.